cmd/pod-scaler/admission.go

```diff
@@ -217,6 +220,9 @@ func useOursIfLarger(allOfOurs, allOfTheirs *corev1.ResourceRequirements, logger
 			if our.Cmp(their) == 1 {
 				logger.Debugf("determined %s %s of %s to be larger than %s configured", field, pair.resource, our.String(), their.String())
 				(*pair.theirs)[field] = our
+				if our.Value() > (their.Value() * 10) {
+					metrics.RecordError("actual memory 10x more than configured amount", promMetrics.ErrorRate)
```

Comment:
we need to be sure to provide information about which workload this is for. You will have to pass that info into this function as it looks like it is not in the scope of the function currently.